### DISCUSSION TITLE
Hi,

### DISCUSSION THREAD

**Discussion by leocarrio** [flypythoncom/python#46] [General]
Need someone strong in python to code black jack for final exam in business school

**Comment by yyx2831** [comment]
> Need someone strong in python to code black jack for final exam in business school

Yes, what exactly needs to be written

**Reply by leocarrio** [reply]
https://replit.com/@kalel1130/intropythonexam

That's de link of the final job 

I think someone good can do it in 10 minutes

**Reply by leocarrio** [reply]
I really need help for this so guys I m looking for kind people ahahhah

**Reply by yyx2831** [reply]
> https://replit.com/@kalel1130/intropythonexam
> 
> 这是最终作业的链接
> 
> 我认为好人可以在10分钟内完成
Ah, I have a wall here, I can’t see the video

**Reply by leocarrio** [reply]
-Deal both user and computer a starting hand of 2 random card values.

-Detect when computer or user has a blackjack. (Ace + 10 value card).

-If computer gets blackjack, then the user loses (even if the user also has a blackjack). If the user gets a blackjack, then they win (unless the computer also has a blackjack).

-Calculate the user's and computer's scores based on their card values.

-If an ace is drawn, count it as 11. But if the total goes over 21, count the ace as 1 instead.

-Reveal computer's first card to the user.

-Game ends immediately when user score goes over 21 or if the user or computer gets a blackjack.

-Ask the user if they want to get another card.

-Once the user is done and no longer wants to draw any more cards, let the computer play. The computer should keep drawing cards unless their score goes over 16.

-Compare user and computer scores and see if it's a win, loss, or draw.

-Print out the player's and computer's final hand and their scores at the end of the game.

-After the game ends, ask the user if they'd like to play again. Clear the console for a fresh start.

**Reply by leocarrio** [reply]
That's the exercice

**Reply by moonwalk47** [reply]
the more understanding i get the better it becomes 
thanks : )

**Comment by leocarrio** [comment]
-Deal both user and computer a starting hand of 2 random card values.

-Detect when computer or user has a blackjack. (Ace + 10 value card).

-If computer gets blackjack, then the user loses (even if the user also has a blackjack). If the user gets a blackjack, then they win (unless the computer also has a blackjack).

-Calculate the user's and computer's scores based on their card values.

-If an ace is drawn, count it as 11. But if the total goes over 21, count the ace as 1 instead.

-Reveal computer's first card to the user.

-Game ends immediately when user score goes over 21 or if the user or computer gets a blackjack.

-Ask the user if they want to get another card.

-Once the user is done and no longer wants to draw any more cards, let the computer play. The computer should keep drawing cards unless their score goes over 16.

-Compare user and computer scores and see if it's a win, loss, or draw.

-Print out the player's and computer's final hand and their scores at the end of the game.

-After the game ends, ask the user if they'd like to play again. Clear the console for a fresh start.

**Reply by leocarrio** [reply]
Can someone help me ?
